web/src/js/mutations/UpdateMissionNotificationMutation.js

```diff
@@ -0,0 +1,38 @@
+import graphql from 'babel-plugin-relay/macro'
```

Comment:
we won't have this feature in legacy so we can remove this file from the web mutations for now.  We'll need it in tab-web so worth copying elsewhere and holding onto it.

Reply:
Ok!